docs/development/authentication.md

```diff
@@ -36,18 +36,17 @@ details worth understanding:
   which shows all emails that the Zulip server has "sent" (emails are
   not actually sent by the development environment), to make it
   convenient to click through the UI of signup, password reset, etc.
-- There's a management command, `manage.py print_initial_password
-  username@example.com`, that prints out **default** passwords for the
+- There's a management command, `manage.py print_initial_password username@example.com`, that prints out **default** passwords for the
```

Comment:
What's the story with these reindentations?  It seems like prettier prefers to create extremely long lines wherever there's an inline code block, which makes the lines outliers in line length.

Reply:
This seems like a bug, so I opened prettier/prettier#11372. Yeah, we could work around it by manually rewrapping to break lines outside of code spans, or we could turn on `proseWrap: "always"` and let Prettier deal with it.